edge-modules/SimulatedTemperatureSensor/docker/windows/arm32v7/Dockerfile

```diff
@@ -1,5 +1,6 @@
 ARG base_tag=1.0.0-windows-arm32v7
-FROM azureiotedge/azureiotedge-module-base:${base_tag}
+ARG base_registry
+FROM ${base_registry}/azureiotedge/azureiotedge-module-base:${base_tag}
```

Comment:
Is using the public base registry not by design?

Reply:
I will confirm with Damon next Monday.